mail/dovecot/Makefile

```diff
@@ -56,15 +61,9 @@ CONFIGURE_ARGS += \
 	--with-notify=dnotify \
 	--without-lzma \
 	--without-lz4 \
-	--with-icu=no
-
-ifneq ($(strip $(CONFIG_DOVECOT_LDAP)),)
-  CONFIGURE_ARGS+= \
-	--with-ldap=yes
-else
-  CONFIGURE_ARGS+= \
-	--with-ldap=no
-endif
+	--with-icu=no \
+$(if $(CONFIG_DOVECOT_LDAP),--with-ldap=yes,--with-ldap=no) \
```

Comment:
Please also move the LDAP change to a separate commit. Preferably before this commit. Thanks.